app/views/LocationTracking.js

```diff
@@ -86,8 +86,12 @@ class LocationTracking extends Component {
       isLogging: '',
       currentState: StateEnum.NO_CONTACT,
     };
-
-    this.checkCurrentState();
+    try {
```

Comment:
If we don't add this try catch then tests don't work as `PERMISSIONS` is not imported in the jest mock